components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Extensions/PublishedDataItemsModelEx.cs

```diff
@@ -31,33 +31,10 @@ public static class PublishedDataItemsModelEx {
                     continue;
                 }
                 var monitoredItem = item.ToMonitoredItem();
-                map.Add(monitoredItem.Id ?? Guid.NewGuid().ToString(), monitoredItem);
-                /*  Heartbeat using triggering mode
-                if (monitoredItem.HeartbeatInterval == null) {
-                    continue;
-                }
 
-                //
-                // We add a timer as heartbeat trigger that samples
-                // server time, but we configure it so it will not be
-                // part of the notifications (Sampling only).
-                //
-                monitoredItem.TriggerId ??= ("heartbeat_" +
-                    monitoredItem.HeartbeatInterval.Value.TotalSeconds.ToString()).ToSha1Hash();
-                if (map.ContainsKey(monitoredItem.TriggerId)) {
-                    continue;
-                }
-                monitoredItem.MonitoringMode = MonitoringMode.Sampling;
-                map.Add(monitoredItem.TriggerId,
-                    new MonitoredItemModel {
-                        MonitoringMode = MonitoringMode.Sampling,
-                        StartNodeId = "i=2258",
-                        SamplingInterval = monitoredItem.HeartbeatInterval,
-                        Id = monitoredItem.TriggerId,
-                    });
-                */
+                map.Add(monitoredItem.StartNodeId, monitoredItem);
```

Comment:
Should we use a LINQ statement here directly on `dataItems.PublishedData` to avoid dictionary creation and work with lists instead ?